lib/matplotlib/projections/polar.py

```diff
@@ -43,27 +43,19 @@ def __str__(self):
                         self._apply_theta_transforms))
 
     def transform_non_affine(self, tr):
-        xy = np.empty(tr.shape, float)
-
-        t = tr[:, 0:1]
```

Comment:
This slice makes `t` 2D; I don't remember why that was important though.

Reply:
I know, but staring at the code it seems unlikely to matter (it's been like that ever since polar.py  appeared in 2008, so probably just "historical").